src/RuleMatchedLog.php

```diff
@@ -220,41 +221,7 @@ public function showItemForm($items_id, $itemtype)
         $iterator = $DB->request($params);
         $number   = $iterator->current()['cpt'];
 
-       // Display the pager
         Html::printAjaxPager(self::getTypeName(2), $start, $number);
```

Comment:
could be migrated to twig with

```twig
{{ call('Html::printerAjaxPager', [
                             call('RuleMatchedLog::getTypeName', [0]), start, number
                           ])|raw }}
```

you may have to adapt my example
perhaps it will be necessary to output ```call('RuleMatchedLog::getTypeName', [0])``` in a dedicated ```set```)